drivers/video/fbdev/bcm2708_fb.c

```diff
@@ -956,12 +1153,12 @@ static int bcm2708_fb_remove(struct platform_device *dev)
 		iounmap(fb->fb.screen_base);
 	unregister_framebuffer(&fb->fb);
 
-	dma_free_writecombine(&dev->dev, SZ_64K, fb->cb_base, fb->cb_handle);
-	bcm_dma_chan_free(fb->dma_chan);
+	dma_free_writecombine(&dev->dev, SZ_64K, cb_base, cb_handle);
```

Comment:
So we created multiple frame buffers, but we only deregister one (the last one created due to the comment above about drvdata).